build_sdk.py

```diff
@@ -742,7 +743,17 @@ def main() -> None:
         root_dir / "bin",
         root_dir / "board",
     ]
-    for board in SUPPORTED_BOARDS:
+
+    if args.board:
+        b_idx = 0;
+        for board in SUPPORTED_BOARDS:
+            if board.name == args.board:
+                boardlist = SUPPORTED_BOARDS[b_idx : b_idx + 1]
+            b_idx += 1
+    else:
+        boardlist = SUPPORTED_BOARDS
+
```

Comment:
I have also thought about adding something like this since it is useful for development as unfortunately when there are a lot of boards/configurations it takes a while to produce the SDK.

We could just have something like `board_list = [board for board in build_sdk.SUPPORTED_BOARDS if board.name == target_board]`

Reply:
(answered in the next comment)